app/static/pages/volunteer/VolunteerInfoDefinitions.js

```diff
@@ -67,18 +67,17 @@ oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
       'impact the lives of many underprivileged children by providing them ' +
       'with free quality education. I am grateful to be a part of this ' +
       'venture.',
-    position: 'Learner Feedback Program Volunteer'
+    position: 'Learner Feedback Program Volunteer, India'
   }, {
-    fullName: 'Yamama Shaka\'a',
-    profilePictureImageFilename: 'yamama.jpeg',
```

Comment:
Should we delete the `yamama.jpeg` file?

Reply:
Done, thanks!